packages/web_pointer_interceptor/README.md

```diff
@@ -0,0 +1,29 @@
+# web_pointer_interceptor
+
+A small Flutter widget to prevent clicks (in web) from being intercepted by underlying HtmlElement views.
+
+## How to use
```

Comment:
I'd add a section preceding this one that explains how it works. Something along these lines:

## How does it work

`PointerInterceptor` creates a platform view consisting of an empty HTML element. The element has the size of its child widget and is inserted in the layer tree behind the child in paint order. When a pointer event lands on the interceptor it is redirected to the Flutter framework. Other platform views that appear behind the interceptor do not receive the pointer event.

Reply:
Yep, good one, I added a couple of little graphics to illustrate the problem and the solution.